model/survey-instance.js

```diff
@@ -41,10 +45,10 @@ function register (sequelize) {
             actualSubmissionTime: {
                 type: Sequelize.DATE
             },
-            surveyInstanceCompleted: {
-                type: Sequelize.BOOLEAN,
-                allowNull: false,
-                defaultValue: false
+            state: {
```

Comment:
Yeah. This will I guess. I will test all the APIs again once we have some rules in the system.